src/lib/config.ts

```diff
@@ -40,6 +40,14 @@ export const config: Config = {
     },
     privacyUrls: {
       cookiePolicyUrl: process.env.OKP4_COOKIE_POLICY_URL
+    },
+    materialsUrls: {
+      technicalDocsUrl: process.env.OKP4_TECHNICAL_DOCS_URL,
+      githubOkp4dUrl: process.env.OKP4_GITHUB_OKP4D_URL,
+      githubContractsUrl: process.env.OKP4_GITHUB_CONTRACTS_URL,
+      githubAwesomeUrl: process.env.OKP4_GITHUB_AWESOME_URL,
+      discordChannelUrl: process.env.OKP4_DISCORD_CHANNEL_URL,
+      discordTicketChannelUrl: process.env.OKP4_DISCORD_TICKET_CHANNEL_URL
```

Comment:
@houssemmimoun27 
It will be better to declare variables into dedicated sub-parts to gather them based upon their purpose, not upon the page they belong to. 

E.G. `technicalDocsUrl` should be part of `docsUrls` object, `discordTicketChannelUrl` should be part of `supportUrls`, etc..

I